### PR TITLE
Scrape sort names

### PR DESCRIPTION
**Pull request by ondenman**
This is a step towards importing the next term. 

The current data contains sort names. This PR adds a ':sort_name' field for each member.

The sort names are captured from the member list page. Each row on the member list page is represented by a `MemberRow` object which contains the member URL (if present) and sort name.

Part of: https://github.com/everypolitician/everypolitician-data/issues/26548

### CODE REVIEW

**Review by tmtmtmtm — CHANGES_REQUESTED** [review]
This is a little hard to follow, and I suspect it's because there's too much going on at once. The PR claims it's only about adding a single field, but there's a lot of other preparatory work happening too. I think it would be better to move those out to separate PRs in the spirit of "make the change easy, then make the easy change".

**Comment by ondenman** [comment]
Closing in favour of: https://github.com/everypolitician-scrapers/croatia-parliament/pull/21
